pkg/service/oidc_provider_test.go

```diff
@@ -0,0 +1,186 @@
+package service
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/google/uuid"
+	v3 "github.com/paralus/paralus/proto/types/commonpb/v3"
+	systemv3 "github.com/paralus/paralus/proto/types/systempb/v3"
+)
+
+func performOidcProviderBasicChecks(t *testing.T, provider *systemv3.OIDCProvider, uuuid string, pruuid string) {
+	if provider.GetMetadata().GetName() != "user-"+uuuid {
+		t.Error("invalid name returned")
+	}
+	t.Log("PROVIDER: ", provider.GetSpec().GetProviderName())
+	if provider.GetSpec().GetProviderName() != "provider-"+pruuid {
+		t.Error("invalid provider name returned")
+	}
+}
+
+func TestOidcCreateProvider(t *testing.T) {
```

Comment:
can you add one more test case for oidc provider create with Duplicate issuer url